[2022-08-22 17:25:15,963] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-22T17:20:00+00:00 [queued]>
[2022-08-22 17:25:16,009] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-22T17:20:00+00:00 [queued]>
[2022-08-22 17:25:16,009] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 17:25:16,009] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-22 17:25:16,009] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 17:25:16,057] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_aguardando> on 2022-08-22 17:20:00+00:00
[2022-08-22 17:25:16,112] {standard_task_runner.py:52} INFO - Started process 89796 to run task
[2022-08-22 17:25:16,145] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_aguardando', 'scheduled__2022-08-22T17:20:00+00:00', '--job-id', '1863', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmp1fyd9o6b', '--error-file', '/tmp/tmpbdb3c4vz']
[2022-08-22 17:25:16,146] {standard_task_runner.py:80} INFO - Job 1863: Subtask popula_elastic_aguardando
[2022-08-22 17:25:16,510] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-22T17:20:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-22 17:25:17,078] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_aguardando
AIRFLOW_CTX_EXECUTION_DATE=2022-08-22T17:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-22T17:20:00+00:00
[2022-08-22 17:25:17,103] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.022s]
[2022-08-22 17:25:19,909] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_aguardando [status:200 request:0.011s]
[2022-08-22 17:25:20,647] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-22 17:25:21,042] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.097s]
[2022-08-22 17:25:21,136] {python.py:173} INFO - Done. Returned value was: None
[2022-08-22 17:25:21,177] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_aguardando, execution_date=20220822T172000, start_date=20220822T172515, end_date=20220822T172521
[2022-08-22 17:25:21,241] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-08-22 17:25:21,248] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 89796. PIDs of all processes in the group: [89796]
[2022-08-22 17:25:21,258] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 89796
[2022-08-22 17:25:21,278] {process_utils.py:75} INFO - Process psutil.Process(pid=89796, status='terminated', exitcode=0, started='17:25:15') (89796) terminated with exit code 0
